tests/operators/test_mssql_to_hive.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from collections import OrderedDict
+
+import pymssql
+from mock import patch, PropertyMock, Mock
+
+from airflow.operators.mssql_to_hive import MsSqlToHiveTransfer
+
+
+class TestMsSqlToHiveTransfer(unittest.TestCase):
```

Comment:
Use `@unittest.skipIf` handling if `pymssql` is not install
```suggestion
@unittest.skipIf(pymssql is None, 'pymssql package not present')
class TestMsSqlToHiveTransfer(unittest.TestCase):
```

Reply:
I like that. Thank you 👍

Reply:
@feluelle You are welcome.